src/configuration/default_configuration.ts

```diff
@@ -0,0 +1,25 @@
+import { IConfiguration } from './types'
+
+export const DEFAULT_CONFIGURATION: IConfiguration = {
+  backtrace: false,
+  dryRun: false,
+  exit: false,
+  failFast: false,
+  format: [],
```

Comment:
I noticed earlier that we now default to the progress formatter?
Could this be part of this default configuration object rather than in code?

Or I may be wrong?

Reply:
We always defaulted to the progress formatter, it just happened elsewhere in the code before. We could add it as a default and have any user-specified ones merged in the normal way but it would be a bit strange. Say the user specifies `--format summary`, the resolved configuration would have `format: ['progress', 'summary']`. The latter one would still take precedence, but still feels strange. What do you think?

Reply:
Oh, ok. Thanks for the explanation 👌